src/moaicore/MOAITouchSensor.cpp

```diff
@@ -322,7 +322,8 @@ void MOAITouchSensor::HandleEvent ( USStream& eventStream ) {
 			}
 		}
 		else {
-			
+			if ( idx == UNKNOWN_TOUCH ) return;
```

Comment:
do we have any clue as to when this is unknown?

Reply:
Not completely sure but in my testing, it seems to be related to starting a swipe gesture while on the app (pause menu) and continue the swipe games side.

On the other hand, the crash on iOS QA reported seems to happen when swiping fast a lot.

I wasn't able to repro the second one but the first one gets fixed with this.

Reply:
interesting, that's good to know. thanks!